Add step to check required attribute

thgaskell opened the pull request:
This PR adds the ability to check if an element has a required property on it.

thgaskell committed:
Add step to check required attribute

jaywon reviewed (approved):
LGTM :white_check_mark:

thgaskell committed:
Normalize phrasing for input fields

thgaskell commented:
I updated the step definition because it didn't seem consistent with the other steps.

Also changed references of "input" to "field" for steps as well.
